spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -546,7 +547,7 @@ def restart_kernel(self):
                         before_prompt=True
                     )
                 else:
-                    # For issue 6235.  IPython was changing the setting of
+                    # For spyder-ide/spyder#6235.  IPython was changing the setting of

```

Comment:
For spyder-ide/spyder#6235.  IPython was changing the setting of ->

For spyder-ide/spyder#6235. IPython was changing the setting of

(remove an extra space between 6235.  IPython)